test-app/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "createjs-accessibility-tester",
+  "version": "1.0.0",
+  "description": "Test and sample webapp for the Createjs Accessibility Module",
+  "main": "src/index.js",
+  "scripts": {
+    "build": "rm -rf dist && NODE_ENV=production webpack",
+    "lint": "eslint --color src",
+    "start": "webpack-dev-server --no-inline --port 8007 --host 0.0.0.0",
+    "start_reload": "webpack-dev-server --port 8007 --host 0.0.0.0",
+    "clobber": "rm -rf node_modules/ && npm cache clean --force"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/CurriculumAssociates/createjs-accessibility-tester.git"
+  },
+  "keywords": [
+    "createjs",
+    "accessibility",
+    "508",
+    "wcag",
+    "aria"
+  ],
+  "author": {
+    "name": "Curriculum Associates",
+    "email": "opensource@cainc.com"
+  },
+  "license": "ISC",
+  "devDependencies": {
+    "babel-core": "6.24.0",
+    "babel-loader": "7.1.4",
+    "babel-eslint": "^10.0.2",
+    "babel-preset-es2015": "6.24.0",
+    "babel-preset-react": "6.24.1",
+    "eslint": "^5.16.0",
+    "eslint-config-airbnb": "^17.1.1",
+    "eslint-formatter-pretty": "^2.1.1",
+    "eslint-friendly-formatter": "^4.0.1",
+    "eslint-import-resolver-node": "^0.3.2",
+    "eslint-loader": "^2.2.1",
+    "eslint-plugin-import": "^2.18.2",
+    "eslint-plugin-jsx-a11y": "^6.2.3",
+    "eslint-plugin-react": "^7.14.3",
+    "file-loader": "1.1.11",
+    "html-loader": "0.5.5",
+    "html-webpack-plugin": "3.0.7",
+    "image-webpack-loader": "4.6.0",
+    "webpack": "4.7.0",
+    "webpack-cli": "3.2.3",
+    "webpack-dev-server": "3.2.1"
+  },
+  "dependencies": {
+    "@curriculumassociates/createjs-accessibility": "file:..",
+    "babel-polyfill": "^6.26.0",
+    "gsap": "^3.8.0",
```

Comment:
looks like you added `gsap` to the original `package.json` @helenxuyang?

Reply:
I was getting lint errors from TimelineMax not being defined in a couple files, and the only way I could figure out how to fix it was to import gsap where TimelineMax is from. I'm not sure how it was working before without gsap though, so maybe there's an alternative way to fix it?

Reply:
There weren't any linting issues because of these lines:

https://github.com/CurriculumAssociates/createjs-accessibility-tester/blob/ebbc30fa54b69c7fd1f749f4992185c549cbad18/.eslintrc.js#L9-L14

which relied on

https://github.com/CurriculumAssociates/createjs-accessibility-tester/blob/ebbc30fa54b69c7fd1f749f4992185c549cbad18/src/index.html#L18-L20

We could remove the dependency and just add an eslintrc that extends the base one and adds those globals in the test-app directory to get back to not needing the dependency.